src/lib.rs

```diff
@@ -594,6 +594,20 @@ mod datetime;
 pub mod format;
 mod round;
 
+/// Constants that can be used by all components.
+pub mod consts {
+    /// Constants of type `f32`.
+    pub mod f32 {
+        /// Number of weeks in a year.
+        pub const WEEK_PER_YEAR: f32 = 52.1775;
```

Comment:
```suggestion
        pub const WEEKS_PER_YEAR: f32 = 52.1775;
```
The comment is correct though :sweat_smile: